.github/workflows/main.yml

```diff
@@ -88,13 +88,13 @@ jobs:
         run: poetry run pip install pytest-github-actions-annotate-failures
 
       - name: Run pytest
-        run: poetry run python -m pytest -p no:sugar -q tests/
+        run: poetry run python -m pytest -n auto -p no:sugar -q tests/
 
       - name: Run pytest (integration suite)
         env:
           POETRY_TEST_INTEGRATION_GIT_USERNAME: ${GITHUB_ACTOR}
           POETRY_TEST_INTEGRATION_GIT_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
-        run: poetry run python -m pytest -p no:sugar -q --integration tests/integration
+        run: poetry run python -m pytest -n auto -p no:sugar -q --integration tests/integration
 
       - name: Get Plugin Version (poetry-plugin-export)
```

Comment:
Should we also use parallelism on `poetry-plugin-export` tests?
Though, similarly to integration tests, they take [~15 seconds](https://github.com/python-poetry/poetry/runs/6694707029?check_suite_focus=true#step:18:9) to run without it, which is already pretty fast, so maybe not worth doing so?

Reply:
Yeah. The overhead didn't seem worth it.

Reply:
And need to make sure those tests work correctly under xdist.